89.md

```diff
@@ -65,7 +65,7 @@ The third value of the tag SHOULD be the platform where this recommendation migh
 
 * `content` is an optional `metadata`-like stringified JSON object, as described in NIP-01. This content is useful when the pubkey creating the `kind:31990` is not an application. If `content` is empty, the `kind:0` of the pubkey should be used to display application information (e.g. name, picture, web, LUD16, etc.)
 * `k` tags' value is the event kind that is supported by this `kind:31990`.
-Using a `k` tag(s) (instead of having the kind onf the NIP-33 `d` tag) provides:
+Using a `k` tag(s) (instead of having the kind onf the `d` tag) provides:
```

Comment:
Typo in this line "kind onf" -> "kind of"

Reply:
Fixed, thanks!